README

```diff
@@ -83,6 +100,17 @@ export USERLAND_ARCHIVES
 
         $ cd (component-dir) ; gmake publish
 
+    If you build several components, and some are required dependencies for the
```

Comment:
The encumbered.mk is a static usecase for really encumbered components. For
my practical usecase it allows to publish test packages whose dependencies
I built and published earlier - so they are not available in real
oi-userland (your internet repo) yet...

--
Typos courtesy of GMail on my Samsung Android
05.04.2016 22:41 пользователь "Alexander Pyhalov" <notifications@github.com>
написал:

> In README
> <https://github.com/OpenIndiana/oi-userland/pull/1806#discussion_r58610346>
> :
>
> > @@ -83,6 +100,17 @@ export USERLAND_ARCHIVES
> >
> >          $ cd (component-dir) ; gmake publish
> >
> > +    If you build several components, and some are required dependencies for the
>
> Makefile has
>
> (WS_TOP)/components/$(ENCUMBERED)components.mk:        # Makefile # $(WS_TOP)/components
>         @echo "Generating component list..."
>         @$(BASS_O_MATIC) --workspace=$(WS_TOP) --components=paths \
>                 | sed -e 's;^$(shell pwd)/;;' \
>                 | sed -f components.ignore \
>                 | sed -e 's;^;COMPONENT_DIRS += ;g' >$@
>
> So setting ENCUMBERED variable breaks components.mk generation. It's
> supposed to be set including $(WS_MAKE_RULES)/encumbered.mk .
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/OpenIndiana/oi-userland/pull/1806/files/a5cf2dfed8ea794180a37e12dfbb69d63d7f7f98#r58610346>
>